src/DataStructures/DataBox/DataBox.hpp

```diff
@@ -297,31 +297,19 @@ template <typename TagList, typename Tag>
 using has_subitems = tmpl::not_<
     std::is_same<typename Subitems<TagList, Tag>::type, tmpl::list<>>>;
 
-template <typename TagList, typename Caller, typename Callee, typename List,
-          typename = std::nullptr_t>
+template <typename TagList, typename ComputeTag>
```

Comment:
A comment explaining what the different parts of this are doing would probably be helpful